sqlpage/sqlpage.js

```diff
@@ -3,24 +3,30 @@
 
 const nonce = document.currentScript.nonce;
 
-function sqlpage_card() {
-    for (const c of document.querySelectorAll("[data-pre-init=card]")) {
-        const source = c.dataset.embed;
-        fetch(c.dataset.embed)
-            .then(res => res.text())
-            .then(html => {
-                const body = c.querySelector(".card-content");
-                body.innerHTML = html;
-                c.removeAttribute("data-pre-init");
-                const spinner = c.querySelector(".card-loading-placeholder");
-                if (spinner) {
-                    spinner.parentNode.removeChild(spinner);
-                }
-                const fragLoadedEvt = new CustomEvent("fragment-loaded", {
-                    bubbles: true
-                });
-                c.dispatchEvent(fragLoadedEvt);
-            })
+function sqlpage_embed() {
+  for (const c of document.querySelectorAll("[data-embed]")) {
```

Comment:
is the query better ?
`document.querySelectorAll("[data-embed]:not([aria-busy=true])")`

Reply:
Yes it's better !